activerecord/lib/active_record/associations.rb

```diff
@@ -1250,6 +1250,9 @@ module ClassMethods
         # [collection.create!(attributes = {})]
         #   Does the same as <tt>collection.create</tt>, but raises ActiveRecord::RecordInvalid
         #   if the record is invalid.
+        # [collection.reload]
+        #   Returns an array of all of the associated objects, forcing a database read.
```

Comment:
@sgrif done here and in the backport https://github.com/rails/rails/pull/30141! ✌️